ffc/codegeneration/integrals_template.py

```diff
@@ -16,6 +16,7 @@
     "cell":
     """
 void tabulate_tensor_{factory_name}(ufc_scalar_t* restrict A, const ufc_scalar_t* w,
+                                    const ufc_scalar_t* c,
```

Comment:
Not specific to this PR, but can we reduce code duplication now that all tabulate function have a common signature?

Reply:
It's not that easy, because the fourth argument `entity_local_index` is named differently in the generated code, as `facet`, `vertex` etc. depending on which version is being called. It doesn't matter for the signature, but they are different for the implementation. It is worth considering how to address in another PR, agreed.